<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaoMonitor</class>
 <widget class="QWidget" name="DaoMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="dataSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Value View Stack</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QListWidget" name="wgtDataList"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Value Information</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="wgtDataInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="wgtInfoTableGroup">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Associated Data</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="margin">
         <number>3</number>
        </property>
        <item>
         <widget class="QTableWidget" name="wgtInfoTable">
          <property name="rowCount">
           <number>2</number>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="wgtDataTableGroup">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Local Variables</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTableWidget" name="wgtDataTable">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Double click to view the data object</string>
         </property>
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="wgtExtraTableGroup">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>VM Instructions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QTableWidget" name="wgtExtraTable">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="rowCount">
          <number>5</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="title">
      <string>Value Data</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTextEdit" name="wgtDataValue">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
